core/domain/html_cleaner.py

```diff
@@ -25,6 +25,20 @@
 
 
 def filter_a(name, value):
+    """Filters the target value from the link.
+    
+    Args:
+        name: The link accepted from the user.
+        value: The value for splitting the link.
+        
+    Raises:
```

Comment:
Drop this section entirely -- no exception is raised; there's only internal logging going on.